utils/kubernetes/kubeutil.py

```diff
@@ -159,7 +177,7 @@ def filter_pods_list(self, pods_list, host_ip):
         pods_list['items'] = filtered_pods
         return pods_list
 
-    def retrieve_json_auth(self, url, auth_token, timeout=10):
+    def retrieve_json_auth(self, url, timeout=10):
         """
         Kubernetes API requires authentication using a token available in
         every pod.
```

Comment:
I'd put a docstring here about certificates options and the fact that the token is ignored in that case